buster/formatters/documents.py

```diff
@@ -50,12 +89,59 @@ def format(
             )
             matched_documents = matched_documents.iloc[:num_preserved_docs]
 
+        documents_str = f"<DOCUMENTS>{documents_str}<\\DOCUMENTS>"
```

Comment:
Can we make `DOCUMENTS` an attribute?

Reply:
sure but then we need to define an "inner_tag" and "outer_tag", e.g. `inner_tag = DOCUMENT` `outer_tag=DOCUMENTS`

Reply:
I'm ok with that

Reply:
fixed